migration-from-4x.md

```diff
@@ -1,66 +1,76 @@
+To migrate the core TinyMCE editor to TinyMCE 5, review the following sections:
 
-```js
-<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5-dev/tinymce.min.js" referrerpolicy="origin"></script>
-```
+* [Cloud Delivery](#clouddelivery).
+* [Editor: Changes to Methods](#editorchangestomethods).
+* [Editor: Changes to Settings](#editorchangestosettings).
+* [Browser support: backward compatibility or quirks mode](#browsersupportbackwardcompatibilityorquirksmode).
 
-### Initialization
+### Cloud Delivery
+
+To use TinyMCE 5 from the Tiny Cloud, include this script in your HTML page. Replace 'no-api-key' with [your API key]({{site.url}}/auth/signup/).
 
-The initialization process of TinyMCE 5.0 is the same as TinyMCE 4.x. The bootstrap process and initialization events all remain the same. It still retains a familiar JSON structure. Some `init` configuration in the TinyMCE version 5.0 has been updated to simplify the configuration options, specifically the configuration items for [UI components]({{site.baseurl}}/ui-components/). Please see the [Quick start]({{site.baseurl}}/quick-start) section for more information on setup.
+```html
+<script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
+```
 
-These changes may impact integrations depending upon the level of customization. Refer to the table below for a general guide for understanding the impact on a specific integration scenario:
+For information on using the latest development and testing builds, see: [Cloud deployment guide - Specify editor & plugin versions]({{site.baseurl}}/cloud-deployment-guide/editor-plugin-version/).
 
-| Customization Level | Description | Impact |
-| ------------------- | ----------- | ------ |
-| Minor | Some custom buttons | No UI fixes required. Updating the `addButton` configuration to TinyMCE 5.0 format. |
-| Moderate | A [dialog]({{site.baseurl}}/ui-components/dialog/#dialoginstanceapi) created using `editor.windowManager.open` configuration objects | Convert TinyMCE 4.x config to TinyMCE 5.0 config. |
-| Major | Completely custom dialogs and extended use of the Modern UI framework | The new TinyMCE 5.0 components may not cover all API use cases. However, Tiny strives to create supported workarounds or if there are sufficient requests, add a new component to resolve the use case. |
+### Editor: Changes to API Methods
 
-> Note: For support related issues such as problems while migrating and a workaround, please contact [support](https://support.tiny.cloud/hc/en-us/requests/new). Alternatively, track developer preview issues on GitHub, [here](https://github.com/tinymce/tinymce/labels/dev%20preview).
+* All TinyMCE 4 API methods for creating UI components have been removed. New methods have been added for TinyMCE 5. For information on creating and customizing UI components, see: [User interface components]({{site.baseurl}}/ui-components/).
+* No core editor methods were removed (such as: `tinymce`, `editor`, `selection`, and `on()`).
```

Comment:
I realise this is how it was worded before, but this seems misleading/confusing, as only one of these are methods (`on()`) and it's more so an editor instance API. `tinymce` is a global and the entrypoint to the TinyMCE API, `editor` doesn't really exist and `selection` is an API available on editor instances.

Reply:
updated line 44 to: ```No core editor API methods were removed.```

Reply:
That's misleading as well though, as we did remove some of the `editor.windowManager` APIs?

Reply:
I'm wondering if we should just delete this line altogether?